cmd/mattermost/commands/command.go

```diff
@@ -53,6 +53,15 @@ var CommandDeleteCmd = &cobra.Command{
 	RunE:    deleteCommandCmdF,
 }
 
+var CommandModifyCmd = &cobra.Command{
+	Use:     "modify [commandID]", //"modify"
+	Short:   "Modify a slash command",
+	Long:    `Modify a slash command. Commands can be specified by command ID.`,
+	Example: `  command modify w16zb5tu3n1zkqo18goqry1je --title MyCommand --description "My Command Description" --trigger-word mycommand --url http://localhost:8000/my-slash-handler --creator myusername --response-username my-bot-username --icon http://localhost:8000/my-slash-handler-bot-icon.png --autocomplete --post`,
+	Args:    cobra.MinimumNArgs(1), //No min args
```

Comment:
Unnecesary and wrong comment. Actually I would prefer to use here cobra.ExactArgs(1), because this receive 1 argument and only one.

Reply:
Removed the comments. Changed MinimumNArgs to ExactArgs